test/test.cabal

```
@@ -0,0 +1,72 @@
+cabal-version:      3.12
```

CI is complaining about this value ("  Errors encountered when parsing cabal file ./test/test.cabal:  test/test.cabal:0:0: error:  Unsupported cabal-version 3.12. See https://github.com/haskell/cabal/issues/4899.). What about setting it to be equal to the other `.cabal` file?
```suggestion
cabal-version:      2.4
```